packages/resto-cohttp-client/resto-cohttp-client.1.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "A minimal OCaml library for type-safe HTTP/JSON RPCs"
+maintainer: "contact@nomadic-labs.com"
+authors: ["Nomadic Labs" "Ocamlpro"]
+license: "MIT"
+homepage: "https://gitlab.com/nomadic-labs/resto"
+bug-reports: "https://gitlab.com/nomadic-labs/resto/issues"
+depends: [
+  "ocaml" {>= "4.10"}
+  "dune" {>= "1.11"}
+  "uri" {>= "1.9.0"}
+  "resto-cohttp" {= version}
+  "cohttp-lwt" {>= "1.0.0"}
```

Comment:
```suggestion
  "cohttp-lwt" {>= "4.0.0"}
```
Due to the pattern matching of the `Permanent_redirect` status code. Can you update the upstream opam file as well?

Reply:
For reference: 
```

#=== ERROR while compiling resto-cohttp-client.1.1 ============================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.10.2 | pinned(https://gitlab.com/nomadic-labs/resto/-/archive/v1.1/resto-v1.1.tar.gz)
# path                 ~/.opam/4.10/.opam-switch/build/resto-cohttp-client.1.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p resto-cohttp-client -j 255
# exit-code            1
# env-file             ~/.opam/log/resto-cohttp-client-7-4cecd3.env
# output-file          ~/.opam/log/resto-cohttp-client-7-4cecd3.out
### output ###
#       ocamlc src/.resto_cohttp_client.objs/byte/resto_cohttp_client__Client.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.10/bin/ocamlc.opt -w -40 -open Resto_cohttp -g -bin-annot -I src/.resto_cohttp_client.objs/byte -I /home/opam/.opam/4.10/lib/base -I /home/opam/.opam/4.10/lib/base/base_internalhash_types -I /home/opam/.opam/4.10/lib/base/caml -I /home/opam/.opam/4.10/lib/base/shadow_stdlib -I /home/opam/.opam/4.10/lib/base64 -I /home/opam/.opam/4.10/lib/bytes -I /home/opam/.opam/4.10/lib/cohttp -I /home/opam/.opam/4.10/lib/cohttp-lwt -I /home/opam/.opam/4.10/lib/fieldslib -I /home/opam/.opam/4.10/lib/lwt -I /home/opam/.opam/4.10/lib/parsexp -I /home/opam/.opam/4.10/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.10/lib/re -I /home/opam/.opam/4.10/lib/resto -I /home/opam/.opam/4.10/lib/resto-cohttp -I /home/opam/.opam/4.10/lib/resto-directory -I /home/opam/.opam/4.10/lib/result -I /home/opam/.opam/4.10/lib/seq -I /home/opam/.opam/4.10/lib/sexplib -I /home/opam/.opam/4.10/lib/sexplib0 -I /home/opam/.opam/4.10/lib/stringext -I /home/opam/.opam/4.10/lib/uri -intf-suffix .ml -no-alias-deps -open Resto_cohttp_client -o src/.resto_cohttp_client.objs/byte/resto_cohttp_client__Client.cmo -c -impl src/client.ml)
# File "src/client.ml", line 296, characters 53-72:
# 296 |         | `Moved_permanently | `Temporary_redirect | `Permanent_redirect -> (
#                                                            ^^^^^^^^^^^^^^^^^^^
# Error: This pattern matches values of type [? `Permanent_redirect ]
#        but a pattern was expected which matches values of type
#          Cohttp__.Code.status_code
#        The second variant type does not allow tag(s) `Permanent_redirect
#     ocamlopt src/.resto_cohttp_client.objs/native/resto_cohttp_client__Client.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.10/bin/ocamlopt.opt -w -40 -open Resto_cohttp -g -I src/.resto_cohttp_client.objs/byte -I src/.resto_cohttp_client.objs/native -I /home/opam/.opam/4.10/lib/base -I /home/opam/.opam/4.10/lib/base/base_internalhash_types -I /home/opam/.opam/4.10/lib/base/caml -I /home/opam/.opam/4.10/lib/base/shadow_stdlib -I /home/opam/.opam/4.10/lib/base64 -I /home/opam/.opam/4.10/lib/bytes -I /home/opam/.opam/4.10/lib/cohttp -I /home/opam/.opam/4.10/lib/cohttp-lwt -I /home/opam/.opam/4.10/lib/fieldslib -I /home/opam/.opam/4.10/lib/lwt -I /home/opam/.opam/4.10/lib/parsexp -I /home/opam/.opam/4.10/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.10/lib/re -I /home/opam/.opam/4.10/lib/resto -I /home/opam/.opam/4.10/lib/resto-cohttp -I /home/opam/.opam/4.10/lib/resto-directory -I /home/opam/.opam/4.10/lib/result -I /home/opam/.opam/4.10/lib/seq -I /home/opam/.opam/4.10/lib/sexplib -I /home/opam/.opam/4.10/lib/sexplib0 -I /home/opam/.opam/4.10/lib/stringext -I /home/opam/.opam/4.10/lib/uri -intf-suffix .ml -no-alias-deps -open Resto_cohttp_client -o src/.resto_cohttp_client.objs/native/resto_cohttp_client__Client.cmx -c -impl src/client.ml)
# File "src/client.ml", line 296, characters 53-72:
# 296 |         | `Moved_permanently | `Temporary_redirect | `Permanent_redirect -> (
#                                                            ^^^^^^^^^^^^^^^^^^^
# Error: This pattern matches values of type [? `Permanent_redirect ]
#        but a pattern was expected which matches values of type
#          Cohttp__.Code.status_code
#        The second variant type does not allow tag(s) `Permanent_redirect
```